### ISSUE TITLE
Windows build script breaks env variables on repeated runs

### ISSUE THREAD

**Issue by mike-myers-tob** [facebook/osquery#3850]
`tools\make-win64-binaries.bat` invokes a Visual Studio .bat file in order to set environment variables for building on Windows. It needs a guard to prevent re-invoking `vcvarsall.bat` though, because on repeated invocations, it will re-append more and more to the `path`, `INCLUDE`, `LIB`, and `LIBPATH` variables until they overflow the length limits, after which the osquery build script won't run until you start over with a fresh cmd prompt.

A possible fix is to add a guard variable that ensures the Visual Studio bat files are invoked only once per session.

**Comment by mike-myers-tob** [comment]
```
:: Set the environment for building with Visual Studio
IF NOT DEFINED ONE_TIME_BUILD_VARS (
  call "%VS140COMNTOOLS%vcvarsqueryregistry.bat" 64bit
  call "%VCINSTALLDIR%vcvarsall.bat" amd64
  SET ONE_TIME_BUILD_VARS=1
)
```

**Comment by muffins** [comment]
This was addressed with #3851
